app/views/report_templates/report_data.html.erb

```diff
@@ -0,0 +1,8 @@
+<% title _("Download generated report") %>
```

Comment:
I wonder if having a full page for downloading process is needed? looks a bit odd
how about using the generator page itself, or showing a modal during the process, likewise importing manifest in katello

Reply:
I have done just that previously, but it has to control the page outside of it and that doesn't feel right. I hope `generate` page to be react page in the feature and this component to be one of the components there.